templates/automotive/index.html

```diff
@@ -1,26 +1,25 @@
 {% extends "automotive/base_automotive.html" %}
 
-{% block title %}Commercial Linux for the automotive sector{% endblock %}
-{% block meta_description %}Canonical is introducing Ubuntu, the most popular commercial-grade and open-source Linux distribution, to the  automotive sector. Ubuntu will accelerate automotive OEMs’ efforts to deliver connected  and autonomous vehicles.{% endblock meta_description %}
+{% block title %}Boost your automotive business with Ubuntu{% endblock %}
+{% block meta_description %}Ubuntu is widely used to reduce time to market in automotive companies, as a platform for data analytics, online services, robotics and machine learning. Infotainment, ADAS and cockpit application development on Ubuntu is faster. New real-time hypervisors enable Ubuntu VMs alongside functional safety RTOS solutions.{% endblock meta_description %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1PVliB92h4ZqexaZV9F5MIcX7aRG6yBAGH1hG7x8UZTw/edit{% endblock meta_copydoc %}
 
 {% block content %}
 
-<section class="p-strip--suru-topped is-bordered">
+<section class="p-strip--suru-bottomed">
   <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Secure Linux for the automotive sector</h1>
-      <p>Leverage the most popular open source operating system to accelerate time to market for your connected vehicles. Ubuntu is the choice platform to develop cockpit infotainment, edge connectivity, mixed-criticality and AI for automotive applications. What&rsquo;s more, Ubuntu Core is designed to enhance functional safety for critical embedded systems. Canonical brings 15 years of experience, enabling and supporting Linux to the automotive sector.</p>
-      <p>Tell us about your moonshot.</p>
-      <p><a href="/internet-of-things/contact-us?product=automotive" class="p-button--positive js-invoke-modal">Get in touch</a></p>
+    <div class="col-4">
+      <h1>The future is <br />beyond electric. <br />It’s electrifying.</h1>
+      <p>Smarter autopilots, infotainment, factories and business models.</p>
+      <p><a class="js-invoke-modal" href="/internet-of-things/contact-us?product=automotive">Let’s talk open source&nbsp;&rsaquo;</a></p>
```

Comment:
this is p-button--positive in the design

Reply:
Nice one :+1: